src/stores/store/eth1/local/local.go

```diff
@@ -406,6 +412,13 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 	return fmt.Sprintf("\x19\x01%s%s", domainSeparatorHash, typedDataHash), nil
 }
 
+// getEIP191EncodedData encodes the given message that can be later recovered
+// with the given validator. following EIP-171 spec version 0x0
+func getEIP191EncodedData(address string, msg []byte) string {
```

Comment:
IMO if this function is called only once and it contains only a single line of code there is not real value in doing this encapulation aside so I would rather place the code itself in `SignEIP191Data`

Reply:
I would say it helps reading the EIP-191 transformation more clearly.
I don't like the get name that much, just kept because of the existing EIP-172 thing already existing